### PR TITLE
[console-iot] Update query to fetch list of projects

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Type of change

- Refactoring

### Description

- Fetch list of projects to display messaging projects and iot projects in project list page.
- Add functionality to enable/disable and delete projects.
### Design
marvel design link - https://marvelapp.com/5gbji22/screen/67563419

image

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: console/console-init/ui/src/framgmentTypes.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (ajaypratap003) [inline console/console-init/ui/src/framgmentTypes.json:1]: I think this file would be better to move in schema folder.

File: console/console-init/ui/src/graphql-module/queries/iot_project.ts [hunk 2]
```diff
@@ -104,8 +104,7 @@ const RETURN_IOT_PROJECTS = (filterObj?: any, queryResolver?: string) => {
 
   if (filterObj) {
     //TODO
-    filter = "`$.kind`='IoTProject'";
-    // filter = FILTER_RETURN_IOT_PROJECTS(filterObj);
+    FILTER_RETURN_IOT_PROJECTS(filterObj);
```
Comment (ajaypratap003) [inline console/console-init/ui/src/graphql-module/queries/iot_project.ts:107]: missing to assign filter variable.
filter=FILTER_RETURN_IOT_PROJECTS(filterObj);

File: console/console-init/ui/src/AppLayout.tsx [hunk 3]
```diff
@@ -24,14 +24,24 @@ import avatarImg from "./img_avatar.svg";
 import "./App.css";
 import { useStoreContext } from "./context-state-reducer";
 import { onServerError } from "./graphql-module";
+import {
+  IntrospectionFragmentMatcher,
+  InMemoryCache
+} from "apollo-cache-inmemory";
+import introspectionQueryResultData from "./types/framgmentTypes.json";
```
Comment (anukritijha) [inline console/console-init/ui/src/AppLayout.tsx:31]: Typo in 'fragment'

File: console/console-init/ui/src/graphql-module/queries/project.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import gql from "graphql-tag";
+
+const FILTER_RETURN_PROJECTS = (filterObject?: any) => {
+  const { projectname, projectType } = filterObject;
```
Comment (anukritijha) [inline console/console-init/ui/src/graphql-module/queries/project.ts:9]: Please revisit the name 'filterObject'. It should give me some information about what filter is it? Is this filter based on name or some other property?

File: console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx [hunk 5]
```diff
@@ -337,6 +316,7 @@ export const ProjectListContainer: React.FC<IProjectListContainerProps> = ({
 
   const projectList: IProject[] = getProjects();
 
+  console.log(projectList);
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx:319]: Please remove the console.

File: console/console-init/ui/src/graphql-module/queries/project.ts (hunk 4 above)
Comment (drcoolsanjeev) [reply console/console-init/ui/src/graphql-module/queries/project.ts:9]: Agreed, I will define a type for filterObject when i will implement the filter section for project list

Comment (anukritijha) [reply console/console-init/ui/src/graphql-module/queries/project.ts:9]: @drcoolsanjeev I was referring to the naming here, not the type.

**Pull request merged**